filament/src/RenderPass.cpp

```diff
@@ -581,15 +581,16 @@ void RenderPass::Executor::recordDriverCommands(backend::DriverApi& driver,
                         CONFIG_MAX_BONE_COUNT * sizeof(PerRenderableUibBone));
             }
 
-            auto morphTargetBuffer = info.morphTargetBuffer;
-            if (UTILS_UNLIKELY(morphTargetBuffer)) {
+            auto morphing = soaMorphing[info.index];
```

Comment:
I would like to avoid this dereference here. We should be able to set things up in a way that the info structure is enough to know if we need to take the if or not.

Reply:
Ok it's done. After that we don't have to increase `PrimitiveInfo` size. Shoud I revert related changes?

Reply:
Yes, definitely -- if we can go back to the old size, absolutely. Thanks.
In your last change, I still see the dereference (soaMorphing[info.index];)?

Reply:
Oh I see... I have also removed that. Now we need more space. :)